cfgov/jinja2/v1/_includes/organisms/video-player.html

```diff
@@ -55,14 +55,12 @@
                     {{- ' o-video-player_video-container__flexible' if not is_fcm else '' }}
                     u-show-on-video-playing">
             <div class="o-video-player_video-actions-container">
-                <div class="o-video-player_video-actions">
```

Comment:
Looks like `o-video-player_video-actions` has styles [here](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/organisms/video-player.less#L205), [here](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/organisms/video-player.less#L234), [here](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/organisms/video-player.less#L272), and [here](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/organisms/video-player.less#L297). 

We probably could strip this div container out, but those styles would need to be migrated to `o-video-player_video-actions-container` and that class would need to be renamed because it's not a container anymore. 

I think this removal is messing with the video on non-featured content module videos. See  https://www.consumerfinance.gov/about-us/blog/new-mortgage-servicing-rule-aims-to-prevent-avoidable-foreclosures-as-protections-expire/

Looks like something is getting messed up there:

This PR
<img width="805" alt="Screen Shot 2021-09-09 at 9 55 53 AM" src="https://user-images.githubusercontent.com/704760/132699168-612841d2-d2c6-43c7-a257-dbdc636c6db8.png">

Prod
<img width="798" alt="Screen Shot 2021-09-09 at 9 56 06 AM" src="https://user-images.githubusercontent.com/704760/132699211-4f8a9bdd-02b0-476b-ac85-e13d27beab52.png">

Reply:
FYI, one hopefully convenient way to easily test this without relying on a production page is the video player template debug view, available locally when you are logged into Wagtail at http://localhost:8000/admin/template_debug/v1/video_player/.

Reply:
"video player template debug view" = Rick Astley royalty machine

Reply:
Thanks for catching this, Ans. This is what it looks like with the PR now:

![image](https://user-images.githubusercontent.com/85513449/132881947-69b6de16-2774-4acf-b5e5-905252c560a3.png)